heliopy/data/custom/units_dict.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
Could the code here go inside `heliopy/data/helpers` instead?

Reply:
Yep moved!